packages/react-core/src/components/InputGroup/examples/InputGroup.md

```diff
@@ -58,6 +58,13 @@ class SimpleInputGroups extends React.Component {
             <SearchIcon />
           </Button>
         </InputGroup>
+        <br />
+        <InputGroup>
+          <TextInput name="textInput12" id="textInput12" type="search" aria-label="search input example" />
+          <Button variant="plain" aria-label="search button for search input">
```

Comment:
Can we change the `aria-label` to "popover for input"? The core example is wrong, too.

```suggestion
          <Button variant="plain" aria-label="Popover for input">
```